R/interpolate.R

```diff
@@ -63,12 +65,22 @@ interpolate_package <- function(
   ...,
   .envir = parent.frame()
 ) {
-  path <- system.file("prompts", path, package = package)
-  interpolate_file(path, ..., .envir = .envir)
+  check_string(path)
+
+  path_pkg <- system.file("prompts", path, package = package)
```

Comment:
FWIW you can also do `mustWork = TRUE`, but since you've already written a better error stick with it.

Reply:
Yeah, I almost did that, but `system.file()` throws a super generic `no file found` error.